src/releasers/node.ts

```diff
@@ -39,9 +38,7 @@ export class Node extends ReleasePR {
     if (pkg.name) this.packageName = pkg.name;
 
     const latestTag: GitHubTag | undefined = await this.gh.latestTag(
```

Comment:
In the cases of releasers with a stable branch name, should `latestTag` also be looking up the stable name first, and falling back to iterating over `findMergedReleasePR`?

Reply:
This was an oversight and is missing in this PR. Finding commits since the last release also needs to be able to find the stable release on the branch.